config/etd.yml

```diff
@@ -65,8 +65,15 @@ custom_nodes:
       name: admin_set_id
       property: "%{work_type}.%{field_name}"
       type: String
-    method: Mapping::AdminSet.unprocessed
-    value_from_node_property: admin_set_id
+    method: Mapping::Admin_Set.lookup_admin_set
```

Comment:
Summary of my latest understanding of the naming convention:
- because the name of the custom_node is admin_set, the mapping file should be named as mapping/admin_set.rb
- the mapping method should be Mapping::AdminSet.lookup_admin_set because the class in admin_set.rb is named 'AdminSet'